src/modules/logger/logger.cpp

```diff
@@ -411,6 +411,59 @@ bool Logger::copy_if_updated_multi(LoggerSubscription &sub, int multi_instance,
 	return updated;
 }
 
+int Logger::add_topics_from_file(const char *fname)
+{
+	FILE		*fp;
+	char		line[80];
+	char		topic_name[40];
```

Comment:
making `topic_name` smaller than line, can lead to an overflow in sscanf. It's unlikely, but you never know what a user might enter.